pyproject.toml

```diff
@@ -0,0 +1,140 @@
+[project]
+classifiers=[
+        "License :: OSI Approved :: BSD License",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
+        "Programming Language :: Python :: 3 :: Only",
+        "Development Status :: 5 - Production/Stable",
+    ]
+
+[tool.poetry]
+name = "parliamentarian"
+version = "1.0.0"
+description = "parliamentarian audits your AWS IAM policies."
+readme = "README.md"
+authors = [
+    "Duo Security <scott@summitroute.com>",
+    "Climate LLC <cloudeng@climate.com>",
```

Comment:
Should we be using the bayer.com domain here?

Reply:
We're under the Climate Corporation's organization, so I'd say no unless there's a compelling reason to do so.